permissions/plugin-content-replace.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "content-replace"
+github: "jenkinsci/content-replace-plugin"
+paths:
+- "org/jenkins-ci/plugins/content-replace"
```

Comment:
Does not match POM content: https://github.com/jenkinsci/content-replace-plugin/blob/master/pom.xml#L11

Please see this repo's readme for how this works.

Reply:
The “groupId” in pom.xml was removed, right?

Reply:
That should work.